workflow/scripts/postprocess_go_enrichment.py

```diff
@@ -25,7 +25,7 @@ def extract_study_items(value):
 
 
 def calculate_sums(parsed_terms):
-    return sum(abs(item["value"]) for item in parsed_terms)
+    return sum(2**abs(item["value"]) for item in parsed_terms).log(base=2)
```

Comment:
This implements the numbered suggestion `2.` in the pull request description:
```suggestion
    number_of_genes=len(parsed_terms)
    return (sum(2**abs(item["value"]) for item in parsed_terms)/number_of_genes).log(base=2)
```
So this normalizes for the number of genes in a GO term.